app/models/reports/repository.rb

```diff
@@ -133,68 +83,6 @@ def reporting_schema_v2?
       }
     end
 
-    memoize def ltfu
-      region_period_cached_query(__method__) do |entry|
-        facility_ids = entry.region.facility_ids
-        Patient.for_reports.where(assigned_facility: facility_ids).ltfu_as_of(entry.period.end).count
-      end
-    end
-
-    memoize def ltfu_rates
-      region_period_cached_query(__method__) do |entry|
-        slug, period = entry.slug, entry.period
-        percentage(ltfu[slug][period], cumulative_assigned_patients[slug][period])
-      end
-    end
-
-    memoize def controlled
-      return controlled_v2 if reporting_schema_v2?
-      science("controlled") do |e|
-        e.use { controlled_v1 }
-        e.try { controlled_v2 }
-      end
-    end
-
-    private def controlled_v1
-      region_period_cached_query(__method__) do |entry|
-        control_rate_query.controlled(entry.region, entry.period).count
-      end
-    end
-
-    private def controlled_v2
-      regions.each_with_object({}).each do |region, hsh|
-        if earliest_patient_recorded_at[region.slug].nil?
-          hsh[region.slug] = {}
-          next
-        end
-        hsh[region.slug] = control_rate_query_v2.controlled_counts(region, range: active_range(region))
-      end
-    end
-
-    memoize def uncontrolled
-      return uncontrolled_v2 if reporting_schema_v2?
-      science("uncontrolled") do |e|
-        e.use { uncontrolled_v1 }
-        e.try { uncontrolled_v2 }
-      end
-    end
-
-    private def uncontrolled_v1
-      region_period_cached_query(__method__) do |entry|
-        control_rate_query.uncontrolled(entry.region, entry.period).count
-      end
-    end
-
-    private def uncontrolled_v2
-      regions.each_with_object({}).each do |region, hsh|
-        if earliest_patient_recorded_at[region.slug].nil?
-          hsh[region.slug] = {}
-          next
-        end
-        hsh[region.slug] = control_rate_query_v2.uncontrolled_counts(region, range: active_range(region))
-      end
-    end
-
     memoize def missed_visits_without_ltfu
```

Comment:
All the missed visits queries can be moved over to v2, but I'm going to that in a follow up PR, as this PR has gotten quite big already